### PR TITLE
add option enables syncing of datasources without their corresponding dimension values

### PR DESCRIPTION
**Pull request by CaprinaeWT**
… dimension values

## Pull request type

Jira Link: [INT-](url)

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other (please describe):

## How to test this PR

`storyblok sync --type datasources  --source <SourceSpaceId> --target <TargetSpaceId> --datasource-disable-dimensions-value-sync  `

The` --datasource-disable-dimension-sync` options can be useful in certain scenarios where the synchronization of dimension values is not necessary or desired. While datasources are commonly used for translations, there may be cases where you only need to sync the structure or metadata of the datasources without including the actual dimension values.

For example, if you have a large dataset with frequently changing dimension values and you want to reduce the sync time or avoid potential conflicts, you may choose to disable the dimension sync. By doing so, you can focus on syncing the datasources themselves while excluding the specific dimension values, which can be later populated or updated separately if needed.

In summary, the option provides flexibility and efficiency by allowing users to selectively sync datasources without the burden of syncing the associated dimension values.

## What is the new behavior?

The expected behavior when using the `--datasource-disable-dimension-sync` option is that the synchronization process will exclude the dimension values associated with the datasources. Only the structure and metadata of the datasources will be synced over to the target space, while the actual values of the dimensions will not be transferred.

This means that the target space will have the same datasources as the source space, including their names, configurations, but the specific values within those datasources' dimensions will not be replicated.

### CODE REVIEW

File: src/cli.js [hunk 1]
```diff
@@ -298,12 +300,16 @@ program
         }
       })
 
+      const filterQuery = filter ? buildFilterQuery(keys, operations, values) : undefined
+      const token = creds.get().token || null
```
Comment (ademarCardoso) [inline src/cli.js]: Please remove this two lines, we don't have the filter query and the token it's already added in the line `294`
